src/app/frontend/deploy/createnamespace.html

```diff
@@ -19,10 +19,18 @@
     <h4 class="md-title" >Create a new namespace</h4>
     <p>The new namespace will be added to the cluster.</p>
     <!-- TODO(maciaszczykm): Missing detailed cluster info and learn more link. -->
-    <form id="namespaceForm" ng-submit="ctrl.createNamespace()">
+    <form name="namespaceForm" ng-submit="ctrl.createNamespace()" novalidate>
       <md-input-container class="md-block">
         <label>Namespace name</label>
-        <input ng-model="ctrl.namespace" required>
+        <input name="namespace" ng-model="ctrl.namespace"
+            md-maxlength="{{ctrl.namespaceMaxLength}}"
```

Comment:
ng-maxlength limits the input field. The user simply cannot type any longer after reaching the limit

md-maxlength shows a small hint below the input (e.g. 7/63) and a error in case the limit is exceeded

I think the md-maxlength is more user-friendly